test/pages/test_qo_section.rb

```
@@ -25,4 +26,38 @@ def test_snake_case_to_title
+        </f>
+      </fb>',
+      'today' => '2024-09-26T04:04:04Z'
+    )
+    scripts = xml.xpath('//script[not(@src)]').map { |s| s.content.strip }.reject(&:empty?)
+    refute_empty(scripts, 'Expected non-empty inline scripts to be generated')
+    scripts.each_with_index do |js, idx|
+      Dir.mktmpdir do |dir|
+        file = File.join(dir, "test_#{idx}.js")
+        File.write(file, js)
+        `node --check #{Shellwords.escape(file)} 2>&1`
+        assert_predicate($CHILD_STATUS, :success?, "JS syntax error in script ##{idx}: #{$CHILD_STATUS}\nContent: #{js}")
```

RuboCop (1.86.1)

[convention] Line is too long. [121/120]

(Layout/LineLength)